gateway-release/home/conf/topologies/homepage.xml

```diff
@@ -60,6 +60,24 @@
          <role>identity-assertion</role>
          <name>Default</name>
          <enabled>true</enabled>
+         <!--
+         <param>
+            <name>haddop.proxyuser.impersonation.enabled</name>
+            <value>false</value>
+         </param>
+         <param>
+            <name>haddop.proxyuser.admin.users</name>
+            <value>*</value>
+         </param>
+         <param>
+            <name>hadoop.proxyuser.admin.groups</name>
+            <value>*</value>
+         </param>
+         <param>
+            <name>hadoop.proxyuser.admin.hosts</name>
+            <value>*</value>
+         </param>
+          -->
```

Comment:
Why is this commented out here, just a convenience? Will these comment survive an admin ui read/save?

Reply:
If impersonation.enabled defaults to false then why do we need to comment them out?

Reply:
Nice catch; I'll fix it soon (the comment remained there from testing).

Reply:
Done.